Analysis/Tasks/PWGDQ/tableReader.cxx

```diff
@@ -531,6 +557,28 @@ struct DQTableReader {
         }
       }
     } // end loop over muon track pairs
+
+    for (auto& [trackBarrel, trackMuon] : combinations(tracks, muons)) {
+      twoTrackFilter = trackBarrel.isBarrelSelected() & trackMuon.isMuonSelected();
```

Comment:
here you ave just one cut for the muon and multiple ones for the electrons. So i have the same comment as in the event mixing task. The bitwise AND has to be changed to a normal AND.